decoder/vaapidecoder_h264.cpp

```diff
@@ -1084,8 +1085,10 @@ bool VaapiDecoderH264::DPB::add(const PicturePtr& picture)
         return output(picture);
     }
 
-    while (isFull())
-        bump();
+    while (isFull()) {
+        if (!bump())
```

Comment:
Why we need check this? It's mean we are full, but we can't bump any buffer? This not reasonable since we should bump buffer at full, else we should lose buffer or something.